kingpin/actors/rightscale/api.py

```diff
@@ -330,7 +330,8 @@ def commit_resource(self, res, res_type, message, params={}):
             The Rightscale Resource itself
         """
         res_id = self.get_res_id(res)
-        params['commit_message'] = message
+        if not params:
+            params = {'commit_message': message}
```

Comment:
I wish this was true .. turns out at the `commit_message` field name is only used on some resources, others use `message`, or even `commit`. F'ing rightscale.